automation/run-tests.sh

```diff
@@ -68,17 +71,31 @@ function install_nmstate {
 }
 
 function run_tests {
-    docker_exec "
-      cd /workspace/nmstate &&
-      pytest \
-        --verbose --verbose \
-        --log-level=DEBUG \
-        --durations=5 \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/libnmstate \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/nmstatectl \
-        --cov-report=html:htmlcov-integ \
-        tests/integration \
-    ${nmstate_pytest_extra_args}"
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "tox_code_style" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e flake8,pylint'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "tox_py27" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e check-py27'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "tox_py36" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e check-py36'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "integ" ];then
+        docker_exec "
+          cd /workspace/nmstate &&
+          pytest \
```

Comment:
The tox.ini also contains a definition to run integration tests for both python 2.7 and 3.6 - why are we defining how to run them differently here? Can we remove the definitions from tox or can we also update it and pass in the values for coverage in there somehow?

Reply:
Seems like this belongs in #233